be/src/exec/pipeline/operator.h

```diff
@@ -141,6 +141,8 @@ class Operator {
     // Which plan node this operator belongs to
     const int32_t _plan_node_id;
     std::shared_ptr<RuntimeProfile> _runtime_profile;
```

Comment:
add comment to explain _common_metrics, _unique_metrics and _runtime_profile

Reply:
ok